packages/terra-docs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terra-docs",
+  "main": "lib/Docs.js",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
We will want to do something like this to prevent this from attempting a release in lerna: https://github.com/cerner/terra-toolkit/blob/main/lerna.json#L5-L7

Reply:
afik setting this to private will already prevent the lerna release

Reply:
I'm right:
>Lerna will never publish packages which are marked as private ("private": true in the package.json).

https://github.com/lerna/lerna/tree/master/commands/publish#usage

Reply:
I added the change to the command... it seems like lerna still tries to version the package even if it's not releasing?